integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html

```diff
@@ -14,15 +14,23 @@
 <ng-template #paymentData let-paymentLink="paymentLink">
   <div>
     <div
-      *ngIf="paymentLink.renderType === RENDER_TYPES.DESTINATION"
+      *ngIf="paymentLink.renderType === RENDER_PATTERN.FULL_PAGE"
       class="cx-payment-link"
     >
-      <a
+      <a class="btn btn-primary" [href]="paymentLink.data">{{
+        'opf.checkout.proceedPayment' | cxTranslate
+      }}</a>
+    </div>
+    <ng-container
+      *ngIf="
+        paymentLink.data && paymentLink.renderType === RENDER_PATTERN.IFRAME
+      "
+    >
+      <iframe
+        class="cx-payment-iframe"
         *ngIf="!paymentLink.destination?.form?.length"
```

Comment:
should we also check url not null (paymentLink.destination?.url) in condition? or iframe tag is smart enough to handle empty value?

Reply:
I added additional check, to not render an empty iframe.